jni/android-main.c

```diff
@@ -117,5 +117,7 @@ void android_main(struct android_app* state) {
     lua_close(L);
 
 quit:
+    LOGE("Destroying Native Activity due to previous errors");
```

Comment:
Seems a bit odd to split it up? :-)

```suggestion
    LOGE("Destroying NativeActivity due to previous errors");
```